src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -152,6 +152,25 @@ export class NetworkAreaDiagramViewer {
         return this.dynamicCssRules;
     }
 
+    private getNodeIdFromEquipmentId(equipmentId: string) {
+        const node: SVGGraphicsElement | null = this.container.querySelector(
+            'nad\\:node[equipmentid="' + equipmentId + '"]'
```

Comment:
may be better without usage of + ?
```suggestion
            `nad\\:node[equipmentid="${equipmentId}"]`
```

Reply:
It works, but I will not change it in order to keep consistency with the rest of the code (which uses the plus sign).